examples/distributed_lock/example.py

```diff
@@ -0,0 +1,45 @@
+# ------------------------------------------------------------
+# Copyright 2021 The Dapr Authors
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ------------------------------------------------------------
+
+from dapr.clients import DaprClient
+import logging
+import warnings
+
+
+def main():
+    # Lock parameters
+    store_name = 'lockstore'  # as defined in components/lockstore.yaml
+    resource_id = 'example-lock-resource'
+    client_id = 'example-client-id'
+    expiry_in_seconds = 60
+
+    with DaprClient() as dapr:
+        logging.info(f'Will try to acquire a lock from "{store_name}" for resource={resource_id}')
+        logging.info(f'This client identifier is "{client_id}"')
+        logging.info(f'This lock will will expire in {expiry_in_seconds} seconds.')
```

Comment:
@mukundansundar what does that comment even mean? For the sample I don't mind a bit more verbose log statements as that also essentially functions as documentation.

Reply:
meant it should be `This lock will expire in {expiry_in_seconds} seconds` instead of `This lock will will expire in {expiry_in_seconds} seconds`
double `will`